usaspending_api/broker/helpers/delete_stale_fabs.py

```diff
@@ -42,7 +42,7 @@ def delete_stale_fabs(ids_to_delete):
         )
     if delete_award_ids:
         # Financial Accounts by Awards
-        faba = 'UPDATE "financial_accounts_by_awards" SET "award_id" = null WHERE "award_id" IN ({});'
+        faba = 'UPDATE "financial_accounts_by_awards" SET "award_id" = null, "linked_date" = NOW() WHERE "award_id" IN ({});'
```

Comment:
Should linked_date be `now()` or `null`?

Reply:
Same question as below:

> Assuming the deletion of last fabs transaction led to the award being deleted?..
> If because the award is being deleted, then that will be covered by the delete-tracking of awards already baked into the nightly delta (I believe such a thing exists, but haven't looked for it).
>
>Since the nighty delta goes after the award records first, if the award is now null, it won't be able to figure out _which_ **award** data in elasticsearch needs to be changed. While a `linked_date` indicates that faba record was changed, we can't discover what award was affected.